flagsmith-on-ecs/terraform/variables.tf

```diff
@@ -0,0 +1,117 @@
+variable "region" {
+  type        = string
+  description = "The AWS region to create resources in."
+}
+
+variable "app_environment" {
+  type    = string
+  default = "dev"
+}
+
+variable "app_name" {
+  type = string
+}
+
+
+# VPC
+
+variable "vpc_cidr" {
+  type        = string
+  description = "CDIR of VPC"
+  default     = "10.0.0.0/16"
+}
+
+# load balancer
+
+variable "health_check_path" {
+  type        = string
+  description = "Health check path for the default target group"
+  default     = "/health/"
+}
+
+# ecs fargate
+
+
+variable "docker_image_url" {
+  type        = string
+  description = "Docker image to run in the ECS cluster"
+}
+
+variable "app_count" {
+  type        = number
+  description = "Number of Docker containers to run"
+  default     = 2
+}
+
+# FIXME
+variable "allowed_hosts" {
+  type        = string
+  description = "Domain name for allowed hosts"
+}
+
+
+# logs
+
+variable "log_retention_in_days" {
+  type    = number
+  default = 7
+}
+
+# specify dev/prod django settings
+variable "settings_module" {
+  type = map(string)
```

Comment:
Not sure what's the logic here as the ECS definition only uses the module specified under the `"dev"` key. Could we
1. Get rid of the map and use plain `string` for this value and
2. Have a default of `app.settings.production` which will be used by 99% of the users anyway?